arm-dependencies.sls

```diff
@@ -0,0 +1,78 @@
+arm-dependencies:
+  pkg.installed:
+    - pkgs:
+      - g++-aarch64-linux-gnu
+      - g++-arm-linux-gnueabihf
+
+/home/servo/bin:
+  file.directory:
+    - user: servo
+    - group: servo
+    - dir_mode: 755
+    - file_mode: 644
+    
+arm64_links:
+  cmd.wait:
```

Comment:
These commands should use the `creates` parameter to avoid being re-executed on subsequent runs. Docs are here: https://docs.saltstack.com/en/2015.5/ref/states/all/salt.states.cmd.html#salt.states.cmd.wait.